pandas/core/_numba/kernels/shared.py

```diff
@@ -2,7 +2,13 @@
 import numpy as np
 
 
-@numba.jit(numba.boolean(numba.int64[:]), nopython=True, nogil=True, parallel=False)
+@numba.jit(
+    # error: Any? not callable
+    numba.boolean(numba.int64[:]),  # type: ignore[misc]
```

Comment:
partial stub does not include numba types so they resolve to Any (and not callable)

@mroeschke not sure of best practice, is it preferable to use the numba types instead of the string representation for production code?

Reply:
Either is recommended in the docs. Strings just have some parsing overhead looking at the code, but probably just fine to use the string representation if it makes typing here easier.

Reply:
I think we could evolve the stubs as needed so will leave as-is for now.